.github/workflows/cmake_linux_x64_no_cuda.yml

```diff
@@ -21,8 +21,11 @@ jobs:
     steps:
     - uses: actions/checkout@v2
       with: 
-        submodules: 'true'
-    
+        submodules: 'false'
```

Comment:
does the submodules task does not work ?

Reply:
Yes for some odd reason, when using the submodule task, vcpkg was unable to fetch the old versions of boost. Its kept throwing an error "git tree object does not exist"